src/scenes/ObservationsRouter/adhoc/AdHocObservationDetails.tsx

```diff
@@ -253,7 +252,10 @@ export default function AdHocObservationDetails(props: AdHocObservationDetailsPr
         id: 'photosAndVideos',
         label: strings.PHOTOS_AND_VIDEOS,
         children: (
-          <PhotosAndVideosTab monitoringPlot={observation?.adHocPlot} isCompleted={!!observation?.completedTime} />
+          <PhotosAndVideosTab
+            monitoringPlot={observation?.adHocPlot}
+            isCompleted={!!observation?.adHocPlot?.completedTime}
```

Comment:
Perhaps a future improvement could be no longer needing `isCompleted` as a prop for `PhotosAndVideosTab` since it's using the completed time on the object already being passed in